### PR TITLE
Copy in-place handling from full.py

### PR DESCRIPTION
**Pull request by JensHeinrich**
See #190

### CODE REVIEW

**Comment by nschloe** [comment]
We could adapt all the CLIs to accept multiple input files, too.

**Comment by JensHeinrich** [comment]
> We could adapt all the CLIs to accept multiple input files, too.

Probably that part of cli options could be centralized?

**Comment by nschloe** [comment]
I don't like `additional_files`. Let's treat all files as input files, and write to stdout unless `-i` is given.

**Comment by JensHeinrich** [comment]
``

> I don't like `additional_files`. Let's treat all files as input files, and write to stdout unless `-i` is given.

But that looses the output file option AND it breaks current users setups

**Comment by nschloe** [comment]
> But that looses the output file option AND it breaks current users setups

doit

Let's bump the major version before a release.

File: src/betterbib/cli/format.py [hunk 1]
```diff
@@ -43,19 +61,23 @@ def _get_parser():
...
     parser.add_argument(
-        "infile",
-        nargs="?",
+        "infiles",
+        nargs="+",
         type=argparse.FileType("r"),
         default=sys.stdin,
         help="input BibTeX file (default: stdin)",
     )
     parser.add_argument(
```
Comment (nschloe) [inline src/betterbib/cli/format.py:70]: Let's just drop this. If the user wants an outfile, she can always use `>`.

**Comment by nschloe** [comment]
Let me know when you think this is ready for review.

**Comment by nschloe** [comment]
> Let's just drop this. If the user wants an outfile, she can always use >.

This.

**Comment by JensHeinrich** [comment]
Done

**Comment by nschloe** [comment]
Tests are failing.

**Comment by JensHeinrich** [comment]
I'm on it.
Not sure what to do about `doi2bibtex` though as it only has an outfile

**Comment by nschloe** [comment]
Same: write to stdout.

**Comment by JensHeinrich** [comment]
The `tests/test_bibtex_title.py` is already broken on main

**Comment by JensHeinrich** [comment]
There is a problem with just writing to stdout though:

```py
E       AssertionError: assert 'Syncing... ━...h = jan,\n}\n' == '%comment{Thi...h = jan,\n}\n'
E         + Syncing... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
E         + 
E         + Total number of entries: 1
E         + Found: 1
E           %comment{This file was created with betterbib v3.5.16.}
E           
E           ...
E         
E         ...Full output truncated (15 lines hidden), use '-vv' to show
```

I just added tests to check the stdout output too and realized there is status output there too

**Comment by JensHeinrich** [comment]
Should I:

- Add the outfile functionality back
- Remove the status outputs when not doing `--in-place`
- Put the status outputs on STDERR (uggly but done similarly by GPG)

**Comment by nschloe** [comment]
> Remove the status outputs when not doing --in-place

This.

**Comment by nschloe** [comment]
Let's not call the argument `quiet` but `verbose`. (And invert its meaning of course.)

File: src/betterbib/tools.py [hunk 2]
```diff
@@ -442,3 +448,38 @@ def filter_fields(data, excludes=None):
...
+    try:
+        data = bibtex.Parser().parse_file(infile)
+        return data
+
+    except Exception as e:
+        print("There was an error when parsing " + infile.name)
+        raise e
+
+
+def bibtex_writer(string: str, outfile, to_file: bool = False):
```
Comment (nschloe) [inline src/betterbib/tools.py:472]: I don't like the arguments of `bibtex_writer`. If `to_file` is false, the `outfile` argument isn't used at all. Better `outfile=None` as default. (Or something along this line.)

Comment (nschloe) [reply src/betterbib/tools.py:472]: The function can also get a more generic name, like `write`.

File: src/betterbib/tools.py [hunk 3]
```diff
@@ -442,3 +448,38 @@ def filter_fields(data, excludes=None):
...
+    try:
+        data = bibtex.Parser().parse_file(infile)
+        return data
+
+    except Exception as e:
+        print("There was an error when parsing " + infile.name)
+        raise e
+
+
+def write(string: str, outfile=None, to_file: bool = False):
```
Comment (nschloe) [inline src/betterbib/tools.py:472]: No `to_file`, please!

**Comment by nschloe** [comment]
LGTM, thanks!

**Pull request merged**